.trunk/trunk.yaml

```diff
@@ -40,7 +40,15 @@ lint:
         - name: lint
           run_when: [cli, monitor, ci]
 actions:
+  definitions:
+    - id: generate-pyi
+      description: Generate Python stubs
+      # Don't use hermetic runtime so we can use nightly cargo
+      run: |
+        cargo run --bin stub_gen
+        mv context-py/context{-,_}py.pyi
```

Comment:
Waffled on just calling these directly in CI or requiring folks to autogen these on `pre-commit`, but I landed on the current approach with a centralized Trunk Action to call